spec/spec-helpers.js

```diff
@@ -1,8 +1,11 @@
+const os = require('os');
 const http = require('http');
+const https = require('https');
 const proc = require('child_process');
-const StateController = require('../lib/state-controller');
+const OSXSupport = require('../lib/support/osx');
```

Comment:
how about only importing the support package for the os currently being run so that you don't have to check for `platform == "win32"` in `windows.js`?